library/tasks/GroupMetricsOutputs.wdl

```diff
@@ -1,15 +1,15 @@
 task GroupQCOutputs {
-  Array[File] picard_row_outputs
+  Array[File?] picard_row_outputs
   Array[File] picard_table_outputs
   File hisat2_stats
   File hisat2_trans_stats
   File rsem_stats
   String output_name
   # Runtime
-  String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.0"
-  String mem = 2
-  String cpu = 1
-  String disk_space = 20 
+  String docker = "quay.io/humancellatlas/secondary-analysis-sctools:mk-unpaired-reads"
```

Comment:
Change this to versioned

Reply:
Okay will do once the new version of sctools is released.